share/completions/p4.fish

```diff
@@ -325,21 +325,21 @@ end
 
 complete -c p4 -n __fish_p4_not_in_command -s b -x -d 'Specifies a batch size (number of arguments) to use when processing a command from a file with the -x argfile option'
 complete -c p4 -n __fish_p4_not_in_command -s c -x -d 'Overrides any P4CLIENT setting with the specified client name'
-complete -c p4 -n __fish_p4_not_in_command -s d -r -d 'Overrides any PWD setting (current working directory) and replaces it with the specified directory'
+complete -c p4 -n __fish_p4_not_in_command -s d -r -d 'Overrides any PWD setting and replaces it with the specified directory.'
 complete -c p4 -n __fish_p4_not_in_command -s I -d 'Specify that progress indicators, if available, are desired. This option is not compatible with the -s and -G options'
```

Comment:
Just as an example of "more aggressive"... this could have been just "show process indicators"